github_actions/lib/dependabot/github_actions/file_fetcher.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "dependabot/file_fetchers"
+require "dependabot/file_fetchers/base"
+
+module Dependabot
+  module GithubActions
+    class FileFetcher < Dependabot::FileFetchers::Base
+      def self.required_files_in?(filenames)
+        filenames.any? { |f| f.match?(%r{\.github/workflows/.+\.ya?ml}) }
```

Comment:
FWIW The expression we use `\A\.github/workflows/[^/]+\.ya?ml\z`.

I don't remember why we chose to support only files in the top `.github/workflows` directory, and not, say `.github/workflows/ci/main.yml`, but that's the current situation.